Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -35,15 +35,21 @@ public class KotlinTokenizer: SwiftTokenizer {
             genericParameterClauseTokens
         ].joined(token: declaration.newToken(.space, " "))
         
-        let signatureTokens = tokenize(declaration.signature, node: declaration)
+        var signatureTokens = tokenize(declaration.signature, node: declaration)
         let bodyTokens = declaration.body.map(tokenize) ?? []
         
-        return [
+        if modifierTokens.contains(where:{$0.value == "override" }) {
```

Comment:
Instead of checking the modifiers directly, I am going to move it to an extension like the `isStatic` because it is a handy check to be used by others in future (and safer to check the type directly rather than the text value)